### ISSUE TITLE
Various SPfx webparts styling incorrect - grid not working - borders around webparts

### ISSUE THREAD

**Issue by rmbw74** [SharePoint/sp-dev-docs#6757]
## Category

- [x] Bug

## Describe the bug

Various spfx webparts have begun to show styling errors. one webpart no longer adhering to ms-grid. Other web parts are showing extra borders around them where they didn't previously. 

## Steps to reproduce
Modern Webpage loads with webparts. 
Webparts are not adhering to style

## Expected behavior

Webpart styling should not be changed. No changes have been made to source code. 

## Environment details (*development & target environment*)

- Windows 10
- Sharepoint Online
- Node JS v 8
- Chrome v88.0.4324.190 , Edge v88.0.705.74 , IE v1909
- **Tooling**: SPfx v1.4 

- **Additional details**: *The more context you can provide, the easier it is (and therefore quicker) to help.*
![2021-02-25 09_03_39-Window](https://user-images.githubusercontent.com/15833201/109172495-74735c00-7748-11eb-972d-6aa4b2e0392e.png)
![2021-02-25 09_04_57-Window border](https://user-images.githubusercontent.com/15833201/109172884-d92eb680-7748-11eb-8138-f825100ed762.png)

### Additional context
Possible relate to or recurrence of [https://github.com/SharePoint/sp-dev-docs/issues/5763](5763)

**Comment by bcameron1231** [comment]
We are seeing this issue as well.

**Comment by xichro** [comment]
Bump. Same on a couple of our tenants. Having to apply "display: flex" on some elements to temporarily fix them.

**Comment by bcameron1231** [comment]
Yup this is occurring in multiple tenants. We have solutions built on 1.6 experiencing this.
 
 ms-SPLegacyFabricBlock is no longer being loaded it appears.

**Comment by DonKirkham** [comment]
Bump. Happening on SPFx 1.7.1 also

**Comment by VesaJuvonen** [comment]
We are absolutely tracking this internally. If anyone has premier support option, please do report using that channel as well as helps on increasing internal awareness.

**Comment by DonKirkham** [comment]
@VesaJuvonen any updates? because I'm seeing the correct DOM structure now.

**Comment by rmbw74** [comment]
> @VesaJuvonen any updates? because I'm seeing the correct DOM structure now. It looks like they added the ms-SPLegacyFabricBlock class to the sp-feature-tag div.

@DonKirkham  - i reverted my code in my dev environment and it all seemed back to normal.

**Comment by DonKirkham** [comment]
@rmba74 Thanks Ray. I saw the same thing late yesterday and a Microsoft SP engineer confirmed that the bug fix was being rolled out. I fixed it earlier in our webpart that was having issues. It only took 2 minutes, but several hours later I realize the original dev didn't check in his latest changes amd I was "fixing" code that was out of date. (facepalm) By the time I had access to the correct code, Microsoft had rolled out the fix to me tenant. #wastedday
